MVC/view/CLIrefactor.py

```diff
@@ -66,11 +66,14 @@ def startGame(self):
                 userInput = input("Input must be between 4 and 15 characters! Please reenter your input: ")
 
             if '!' not in userInput:
-                #controller user guess function
+            # controller user guess function
                 if self.controller.controllerGetPuzzleState() != 1:
-                     print("Make sure to start a puzzle before guessing")
+                    print("Make sure to start a puzzle before guessing")
```

Comment:
Maybe something like "Try using the !newpuzzle command to start playing, or !exit to quit." It's not like the user would be 'trying' to guess when there is no puzzle to guess from, it's more likely they didn't read through the intro text or are unsure for whatever reason what to do. Using my suggesting I think would be more direct to give them the two options to either actually start playing or just leave all together.